styles.css

```diff
@@ -213,29 +205,30 @@ border: 2px solid #FFF;
 
 .new-activity-form {
   display: flex;
-  flex-direction: column;
+  justify-content: center;
   background-color: #46424D;
   position: relative;
   height: 80%;
   width: 100%;
   box-shadow: 7px 7px #26222D;
+  border-radius: 5px;
 }
 
 .new-activity-button {
-  align-self: center;
+  align-self: bottom;
   margin-top: 100px;
-  width: 300px;
-  height: 35px;
+  width: 40%;
+  height: 6%;
```

Comment:
As a general rule, heights should be static values, not dynamic ones like percentages. Can lead to unexpected behavior when changing screen sizes

Reply:
Good catch! I don't think we have enough time to change this now but will definitely keep it in mind for future projects.